platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -248,6 +249,22 @@ public int[] screenGetUsableRect() {
 		return result;
 	}
 
+	public int[] screenGetInactiveRect() {
+		int[] result = { 0, 0, 0, 0 };
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+			WindowInsets insets = activity.getWindow().getDecorView().getRootWindowInsets();
+			DisplayCutout cutout = insets.getDisplayCutout();
+			if (cutout != null) {
+				Rect boundingRect = cutout.getBoundingRects().get(0);
```

Comment:
As raised by @akien-mga in his [comment](https://github.com/godotengine/godot/pull/54290#issuecomment-953560422), you're only returning the first cutout. So this approach won't work when there are more than one cutout.